core/backend/src/ChangedElementsDb.ts

```diff
@@ -109,11 +109,7 @@ export class ChangedElementsDb implements IDisposable {
   /** Processes a range of changesets and adds it to the changed elements cache
    * @param requestContext The client request context
    * @param briefcase iModel briefcase to use
-   * @param startChangesetId Start Changeset Id
-   * @param endChangesetId End Changeset Id
-   * @param filterSpatial [optional] Whether to do processing filtering out spatial elements, defaults to false
-   * @param rulesetDir [optional] Directories string for ruleset directory locater
-   * @param tempDir [optional] Directory to use to store temporary Db used to do processing. This Db is cleaned up automatically unless the process crashes.
+   * @param options Options for processing
```

Comment:
Small thing but the list of `@params` in your documentation doesn't match those in your function declaration.